dask/bytes/tests/test_s3.py

```diff
@@ -556,8 +561,10 @@ def test_parquet_append(s3, engine, s3so):
     dd = pytest.importorskip("dask.dataframe")
     pd = pytest.importorskip("pandas")
     np = pytest.importorskip("numpy")
-    if dd._dask_expr_enabled():
-        pytest.skip("need convert string option")
```

Comment:
I'm assuming we're removing this because this functionality exists now

Reply:
This wasn't fixed (different root cause), but have a PR up now that will fix it